storage/replica_test.go

```diff
@@ -1547,6 +1548,10 @@ func TestRangeResponseCacheStoredError(t *testing.T) {
 // can be committed/aborted before being heartbeat.
 func TestEndTransactionBeforeHeartbeat(t *testing.T) {
 	defer leaktest.AfterTest(t)
+	// Don't automatically GC the Txn record: We want to heartbeat the
+	// committed Transaction and compare it against our expectations.
+	// When it's removed, the heartbeat would recreate it.
```

Comment:
`Heartbeat` will change in upcoming work to avoid that (same as `PushTxn`). We won't create those orphan stray txns and even if we did, it wouldn't have intents so nobody would ever push it.